cli/lib/tasks/get-folder-size.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('../fs')
+const { join } = require('path')
+const Bluebird = require('bluebird')
+
+async function getSize (path) {
```

Comment:
Could you add JSDoc comment to this function?

Reply:
Done.